networking/synthetic/stac/script.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/sh
+
+numa_cores=<range>
+mtu_size="1800"
+
+start() {
+    echo "200" > /proc/sys/vm/nr_hugepages
```

Comment:
Move the hugepages allocation to tuned please.

Reply:
ok, will change that